### PR TITLE
Add gui page to view registered clusters

### PR DESCRIPTION
**Pull request by yume-ind**
This PR contains the first page of a gui where the user can view their Cassy clusters on a formatted interface.

Please check the instructions detailed in the README.md to run the server in development mode.

### CODE REVIEW

File: gui/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+```
+npm install
+```
+
+### To run the GUI with Cassy
+See the [Getting Started](../docs/getting-started.md) to set up your Cassy configuration and run the server.
+
+Once you have started Cassy, go to the root folder and run the following command to start the HTTP/1.1 server.
+```
+./entrypoint -dev
```
Comment (feeblefakie) [inline gui/README.md]: I think this is slightly better. (having `dev` and `prod` maybe ?)
```suggestion
./entrypoint -mode=dev
```

Comment (yume-ind) [reply gui/README.md:13]: In its current state, the default mode is `prod`, so you can run the file by just doing `./entrypoint` . Would you still prefer the configuration above?

File: gui/src/components/Clusters.vue [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<template>
+    <div class="container">
+        <h4 class="mt-3">View Clusters</h4>
```
Comment (feeblefakie) [inline gui/src/components/Clusters.vue:3]:
```suggestion
        <h4 class="mt-3">Clusters</h4>
```

File: gui/src/components/Clusters.vue [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        <table class="table table-borderless table-striped">
+            <thead>
+            <tr>
+                <th scope="col">#</th>
```
Comment (feeblefakie) [inline gui/src/components/Clusters.vue:7]: Maybe, we don't need this.

**Comment by feeblefakie** [comment]
@yume-ind Regarding UI/UX, it would be great if you can make it look roughly like reaper's UI so that our tools around C* look consistent and similar, and minimize redundant work for adjusting UI.

**Pull request merged**